src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/azuredevops/model/CommentPosition.java

```diff
@@ -0,0 +1,29 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model;
+
+import java.io.Serializable;
+
+public class CommentPosition implements Serializable {
+    private Integer line;
+    private Integer offset;
+
+    public CommentPosition() {};
+
+    public CommentPosition(Integer line, Integer offset){
+        this.line = line;
+        this.offset = offset + 1;
```

Comment:
Why `+ 1`?

Reply:
line numbering in Azure PullRequest starts at "0", while in SonarQube starts at "1"